lib/export-task.js

```diff
@@ -18,6 +18,30 @@ class ExportTask {
       this[key] = content[key];
   }
 
+  async retry() {
```

Comment:
Add some documentation to the function (similar to the other functions in the file)